src/app/modules/item/pages/item-by-id/item-by-id.component.html

```diff
@@ -131,19 +159,56 @@
           </ng-template>
 
         </ng-container>
-        <alg-item-progress
-          *ngIf="progressTab?.isActive"
-          [itemData]="itemData"
-          [savingAnswer]="(savingAnswer$ | async) ?? false"
-          (skipSave)="skipBeforeUnload()"
-        ></alg-item-progress>
 
         <alg-item-task-edit
           *ngIf="taskEditTab?.isActive"
           [editorUrl]="editorUrl"
           (redirectToDefaultTab)="navigateToDefaultTab(itemData.route)"
         ></alg-item-task-edit>
 
+        <ng-container *ngIf="!!historyTab?.isActive || !!chapterGroupProgressTab?.isActive || !!chapterUserProgressTab?.isActive">
+          <ng-container *ngrxLet="isWatching$ as isWatching">
+            <ng-container *ngIf="(!isWatching && (itemData.item.permissions | allowsViewingContent)) || (itemData.item.permissions | allowsWatchingResults) ; else unallowed">
+              <div *ngIf="(savingAnswer$ | async) ?? false" class="save-answer-loader">
+                <p class="save-answer-loader-message">
+                  <span i18n>Saving answer before loading submission...</span>
+                  <alg-loading size="medium"></alg-loading>
+                  <p-button type="button" (click)="skipBeforeUnload()" i18n-label label="Skip"></p-button>
+                </p>
+              </div>
+
+              <alg-item-log-view
+                *ngIf="!!historyTab?.isActive"
+                [itemData]="itemData"
+              ></alg-item-log-view>
+
+              <alg-chapter-group-progress
+                *ngIf="!!chapterGroupProgressTab?.isActive"
+                [itemData]="itemData"
+              ></alg-chapter-group-progress>
+
+              <div *ngIf="!!chapterUserProgressTab?.isActive">
+                <p class="alg-h5 alg-secondary-color stats-title" i18n>
+                  This tab gives details about the date, time and number of attempts for each item in this chapter.
+                </p>
```

Comment:
Remove this text for now, that was just an example (and the sentence is not great)

Reply:
Fixed #1365